imports/ui/case-explorer/case-explorer.jsx

```diff
@@ -154,12 +154,12 @@ class CaseExplorer extends Component {
           sortBy={sortBy}
           roles={['All', 'Assigned to me']}
           labels={[
-            [SORT_BY.DATE_DESCENDING, {category: 'Newest (Created)', selected: 'Created ↓'}],
-            [SORT_BY.DATE_ASCENDING, {category: 'Oldest (Created)', selected: ' Created ↑'}],
+            [SORT_BY.DATE_DESCENDING, {category: 'Created - Newest', selected: 'Created ↓'}],
+            [SORT_BY.DATE_ASCENDING, {category: 'Created - Oldest', selected: ' Created ↑'}],
             [SORT_BY.NAME_ASCENDING, {category: 'Name (A to Z)', selected: 'A to Z ↑'}],
             [SORT_BY.NAME_DESCENDING, {category: 'Name (Z to A)', selected: 'Z to A ↓'}],
-            [SORT_BY.LATEST_UPDATE, {category: 'Latest (Updated)', selected: 'Updated ↓'}],
-            [SORT_BY.OLDEST_UPDATE, {category: 'Oldest (Updated)', selected: 'Updated ↑'}]
+            [SORT_BY.LATEST_UPDATE, {category: 'Updated - Latest', selected: 'Updated ↓'}],
```

Comment:
@sophieyang0813 I really prefer that we use 
category: 'Updated - Newest' instead of 'Updated - Latest'.

I have 2 reasons for that:
---> this is consistent with the wording for "Created"
---> I'm personally confused and have to think about the differenc between "Latest" and "Oldest" so I'm guessing that other user might be too